src/hamiltonian.jl

```diff
@@ -138,15 +138,32 @@ phasepoint(
     h::Hamiltonian
 ) where {T<:Real} = phasepoint(h, θ, rand(rng, h.metric))
 
+abstract type AbstractRefreshment end
```

Comment:
`AbstractRegreshment` is a bit vague - it's a bit hard to understand what it means. Maybe rename this to `AbstractMomentum`?